Quantum-Control-Applications/Superconducting/Multiple Flux Tunable Transmons/Use Case 2 - Two Qubit Randomized Benchmarking/configuration.py

```diff
@@ -0,0 +1,562 @@
+config = {
+    "version": 1,
+    "controllers": {
+        "con1": {
+            "type": "opx1",
+            "analog_outputs": {
+                1: {"offset": 0.0, "delay": 0},
+                2: {"offset": 0.0, "delay": 0},
+                7: {
+                    "offset": 0.20529425782807967,
+                    "delay": 0,
+                    "crosstalk": {7: 0.9997004946118614, 8: -0.016448562428420136},
+                    "filter": {"feedforward": [1.0], "feedback": [0.0]},
+                },
+                5: {"offset": 0.0, "delay": 0},
+                6: {"offset": 0.0, "delay": 0},
+                3: {"offset": 0.0, "delay": 0},
+                4: {"offset": 0.0, "delay": 0},
+                8: {
+                    "offset": -0.2743486982547745,
+                    "delay": 0,
+                    "crosstalk": {7: 0.018203152157770183, 8: 0.9997004946118613},
+                    "filter": {
+                        "feedforward": [1.1359055538996878, -1.03667045777435],
+                        "feedback": [0.9007649038746622],
+                    },
+                },
+            },
+            "analog_inputs": {
+                1: {"offset": 0, "gain_db": 0},
+                2: {"offset": 0, "gain_db": 0},
+            },
+            "digital_outputs": {},
+        }
+    },
+    "elements": {
+        "qubit0_xy": {
+            "mixInputs": {
+                "I": ["con1", 1],
```

Comment:
Interesting, I thought that it has to be a tuple: ("con1, 1)"

Reply:
Interesting to me as well :) 
I think that tuple and list can be used interchangeably in the config, which is a bit annoying